src/tasks/package.js

```diff
@@ -74,6 +84,23 @@ module.exports = (config) => {
         'ci:lint': 'npm run lint && npm run security',
         'ci:test': 'npm run test -- --runInBand',
         'ci:coverage': 'npm run test:coverage -- --runInBand',
+        'webpack-defaults': 'webpack-defaults',
+      },
+      dependencies: existingPackage.dependencies || {},
+      devDependencies: existingPackage.devDependencies || {},
+      engines: {
+        node: `>= ${config.maintLTS} || >= ${config.activeLTS}`,
+      },
+      peerDependencies: {
+        webpack: `^${config.maintWebpack} || ^${config.activeWebpack}`,
+      },
+      homepage: `https://github.com/${repository}`,
```

Comment:
```
// README on webpack.js.org, but the user is directly on the docs page instead
homepage: `https://webpack.js.org/loaders/${name}`,
```

Reply:
⚠️  Needs to be handled differently between `loaders/plugins`

### `Loaders`

```
`https://webpack.js.org/loaders/${name}`
```
### `Plugins`

```
`https://webpack.js.org/plugins/${name}`
```

Reply:
This is why I just left it set to the repo for the time being. Can be when the CLI wrapper adds context which I wasn't going to do until we have the important stuff out of the way.